R/roxygen.R

```diff
@@ -9,7 +9,8 @@
 #' @param lockfile Path to a lockfile. When `NULL` (the default), the `renv.lock` located in the root of the current project will be used.
 #'
 #' @param library The \R library to be used. When `NULL`, the active project
-#'  library will be used instead.
+#'  library will be used instead. If no project is active, will fall back
+#'  to the system library.
```

Comment:
I wonder if this could just be something like "When `NULL`, the current library paths will be used."? IIRC almost all places in `renv` just use the current library paths, with the assumption that `load()` would have set up the default library paths.